cvat/apps/engine/static/engine/js/annotationUI.js

```diff
@@ -547,6 +547,13 @@ function buildAnnotationUI(jobData, taskData, imageMetaData, annotationData, ann
                 return `${window.location.origin}/?${this.value}`;
             },
         },
+        frameClipper: {
+            clipToFrame: false,
+
+            clamp(value, min, max) {
+                return this.clipToFrame ? value : Math.clamp(value, min, max);
```

Comment:
by the way, the logic is reversed here: you do clip when `clipToFrame` is `false`

Reply:
Fixed it and also addressed previous comment regarding simplification.